skiko/build.gradle.kts

```diff
@@ -268,7 +268,7 @@ tasks.withType(CppCompile::class.java).configureEach {
             compilerArgs.addAll(
                 listOf(
                     "-fno-rtti",
-                    "-fno-exceptions",
+                    "-fexceptions",
```

Comment:
Do not think it's a good idea, let's avoid using exceptions.

Reply:
without it the linux version won't compile

Reply:
We shalln't try/catch in non-Windows code

Reply:
Done.